src/include/migraphx/matcher.hpp

```diff
@@ -895,9 +895,9 @@ inline auto has_value(T x, std::size_t atol_mult = 10, std::size_t rtol_mult = 1
             }
             else
             {
-                auto eps = std::numeric_limits<type>::epsilon();
+                float eps = std::numeric_limits<type>::epsilon();
```

Comment:
This should be double if you dont use `auto`.

Reply:
Reverted